ahcore/models/feature_extractors.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+
+import torch
+from torch.jit import ScriptModule
+
+from ahcore.models.embedding_functions.vit_embed import ViTEmbed
+from ahcore.models.jit_model import BaseAhcoreJitModel
+from ahcore.utils.types import OutputModeBase, ViTEmbedMode
+
+
+class DinoV2JitModel(BaseAhcoreJitModel):
```

Comment:
I think it would be better to just expect any jitted model and not necessarily have a specific class for each different model. The user can jit in a lot of ways and planning for all of them would be a pain. I think it should be up to the user to jit the correct model and the jitted model should give the expected output. Maybe checking if the output is as expected should be implemented in the base class